Cutout.Docs/articles/file-templates.md

```
@@ -0,0 +1,42 @@
+# File Templates
+
+The templates can also be stored in external files.
+
+To get started, create a '.editorconfig' file in your project.
+
+In this, connect static files to valid methods in your project.
+
+For example, if you have a template called 'my-template.txt', 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)